flow360/component/flow360_params/validations.py

```diff
@@ -22,6 +22,19 @@
 from .volume_zones import HeatTransferVolumeZone
 
 
+def _ignore_velocity_type_in_boundaries(values):
+    """values here is actually json dict."""
+    for boundary_name, obj in values.items():
+        if "velocityType" in obj:
+            log.warning(
+                f"In {boundary_name}: 'velocityType' is no longer supported and will be ignored."
```

Comment:
Can you change it to this message to be consistent with the solver:
f"Specifying velocityType for boundary condition {bcType} is no longer supported. The boundary velocity type \
must now always be prescribed relative to to the inertial reference frame."

Reply:
Done